### DISCUSSION TITLE
How to Unittest a custom log processor

### DISCUSSION THREAD

**Discussion by HugzM** [open-telemetry/opentelemetry-dotnet#4965] [Q&A]
I am using a custom Log Processor inheriting form BaseProcesor<Logrecord>. I want to setup a Logrecord so I can test my processing but Logrecord does not have accessible Ctor ( all internals). How can I tets my class since all method signature needs a logRecord?

**Comment by cijothomas** [comment]
Can you use InMemoryExporter to get a hold of LodRecords? This repo should have plenty of examples in the tests.
